etc/openstack_deploy/templates/openstack_user_config.yml.j2

```diff
@@ -503,7 +503,7 @@ storage_hosts:
           volume_group: cinder-volumes
           volume_driver: cinder.volume.drivers.rbd.RBDDriver
           volume_backend_name: rbd
-          rbd_pool: cinder_volumes
+          rbd_pool: volumes
```

Comment:
Why do you want to change rbd_pool name cinder_volumes by volumes? IMO cinder_volumes is better explanatory and should not be changed.

Reply:
ceph-ansible configures it this way (https://github.com/ceph/ceph-ansible/blob/04ec1ad3cc0d4b60458a8b14dcc0b640e80f9eef/tests/functional/podman/group_vars/all)

Reply:
It's a just a name example. IMO, 'cinder-volumes' as opposed to just 'volumes' is more self-explanatory because you can tell by the name that the pool is about cinder and not something else. I prefer it this way.

Reply:
The problem is that cinder wont create the rdb pool and if you want to use cinder_volumes as the rdb pool name, you must also change the ceph configuration showed in the link.

Reply:
Okay but that's not a problem of name. It used to work... I think the problem is more about that volume_group: cinder-volumes and rbd_pool: cinder_volumes no  longer match. There has been a clunky change at some point there :(  Check https://docs.openstack.org/openstack-ansible-os_cinder/latest/configure-cinder.html#configuring-cinder-to-use-ceph
And working example at https://www.openstackfaq.com/openstack-ansible-ceph/
In summary, I think the correction is to define rbd_pool as cinder**-**volumes (with an hyphen)